### ISSUE TITLE
Updating OC to a newer version?

### ISSUE THREAD

**Issue by LukeGotBored** [xWeapon/CoreBook_X_OC#13]
Are you ever going to update this repo to a newer version of opencore? I'm running into a few issues (Volume not showing up on boot) on this current configuration (Chuwi Corebook X Gen-1)

**Comment by rboldini** [comment]
TL;DR 

My battery died and the PS (65W) doesn't supply sufficient energy to keep it power on.

Delivery is programmed for FEB/MAR 2023.

You can try solving the issue updating NVmEFix.kext or try updating OC following both my guide and Dortania Guide.

I followed also the intel Wi-Fi driver source deployment, in this months, from zxy and it seems to be lot better than the version provided in this repo.

--------------------------------

This repo unfortunately will be probably deprecated in mid 2023, cause  I'm not using Corebook anymore, having different laptops for work purpose, and one of this is a Real MacBook.

So the use of this repo is anyways discouraged. Must be better to know how to produce a personal OC configuration.

I hope to have time to update it, knowing that lot of people effectively uses my patches.

**Comment by LukeGotBored** [comment]
I'll try to write a custom config for myself, if I actually manage to get something decent, I'll probably either make a pull request or release it on my own account 👍 
Thanks anyways!

**Comment by leonardotoro** [comment]
Hi all,
I've been circling the problem for days. First I tried to do everything myself by following Dartana's guide but no dice, I always encounter the error afps_module_start:2485 which doesn't let me get going, then in a moment of desperation I copied the entire EFI folder to use the code that was entered here and to my surprise, the pc doesn't boot, reboots and I'm forced to start windows.

What version of OpenCore are you using? Could it be that or the fact that I use Big Sur from Recovery?

**Comment by LukeGotBored** [comment]
Hi there! I unfortunately stopped working on the EFI since it completely stopped working after a few weeks, I could probably share it either here or in a new repo (although the battery still refuses to show up), though just to be sure, are you 100% sure you're using a gen 1 Corebook x?
